crates/wasi-nn/src/backend/openvino.rs

```diff
@@ -157,24 +148,51 @@ impl From<SetupError> for BackendError {
 
 /// Return the execution target string expected by OpenVINO from the
 /// `ExecutionTarget` enum provided by wasi-nn.
-fn map_execution_target_to_string(target: ExecutionTarget) -> &'static str {
+fn map_execution_target_to_string(target: ExecutionTarget) -> DeviceType<'static> {
```

Comment:
If possible, let's rewrite this as `impl From<ExecutionTarget> for DeviceType<'static>`. If not possible then at least change the function name and documentation.